server/events/project_locker.go

```diff
@@ -63,7 +63,7 @@ func (p *DefaultProjectLocker) TryLock(log *logging.SimpleLogger, pull models.Pu
 	}
 	if !lockAttempt.LockAcquired && lockAttempt.CurrLock.Pull.Num != pull.Num {
 		failureMsg := fmt.Sprintf(
-			"This project is currently locked by #%d. The locking plan must be applied or discarded before future plans can execute.",
+			"This project is currently locked by #%d. The locking plan must be applied or discarded before future plans can execute. Comment with `atlantis plan` on this PR, once the lock is released.",
```

Comment:
I'd like to change this so it looks like:

**Unable to run `plan`**. This project is currently locked by an unapplied plan from pull #2. To continue, delete the lock from #2 or apply that plan and merge the pull request.

Once the lock is released, comment `atlantis plan` here to re-plan.


The full string will be:
```
fmt.Sprintf("**Unable to run `plan`**. This project is currently locked by an unapplied plan from pull #%d. To continue, delete the lock from #%d or apply that plan and merge the pull request.\n\nOnce the lock is released, comment `atlantis plan` here to re-plan.", lockAttempt.CurrLock.Pull.Num, lockAttempt.CurrLock.Pull.Num)
```

Reply:
cool, updated!